aidants_connect_web/templates/aidants_connect_web/managment/notify_soon_expired_mandates.html

```diff
@@ -0,0 +1,41 @@
+{% extends "layouts/email_base.html" %}
+{% block email_body %}
+  <table border="0" cellpadding="0" cellspacing="0" class="body">
+    <tr>
+      <td>&nbsp;</td>
+      <td class="container">
+        <div class="content">
+
+          <!-- START CENTERED WHITE CONTAINER -->
```

Comment:
En règle générale je préfère éviter de reporter ce genre de commentaire dans le HTML, et utiliser plutôt le commentaire du langage de gabarit : 
```suggestion
          {# START CENTERED WHITE CONTAINER #}
```

Surtout pour du HTML qui se retrouve ensuite dans du webmail.

Reply:
J'ai juste reprise le template existant mais tu as complètement raison.